e2e/pages/home.page.ts

```diff
@@ -0,0 +1,21 @@
+import { Page, Locator } from '@playwright/test';
+
+export class HomePage {
+  readonly page: Page;
+  readonly activityCard: Locator;
+  readonly paudActivity: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.activityCard = page.locator('h3.text-lg.font-medium');
```

Comment:
I am a proponent of writing tests that imitate how the users use the app.
In this case, the user won't look for any `h3` elements with `text-lg font-medium` class.
So let's avoid this kind of selector.

Instead, you can use Playwright's [new APIs inspired by Testing Library](https://twitter.com/diegohaz/status/1578567655493193728). But first, you need to upgrade the Playwright version to [1.27.0](https://github.com/microsoft/playwright/releases/tag/v1.27.0) to use it.

Reply:
noted with thanks @zainfathoni 
I'll try to use this one [new APIs inspired by Testing Library](https://twitter.com/diegohaz/status/1578567655493193728)